roles/repository/tasks/main.yml

```diff
@@ -401,10 +401,39 @@
     insertafter: EOF
     block: "{{ lookup('file', role_path + '/../../configuration_files/alfresco-global.properties') }}"
   notify:
-    - acs-monitored-restart
+    - restart-alfresco-content
   tags:
     - molecule-idempotence-notest # see OPSEXP-1280
 
+- name: Create monitoring start up service
+  template:
+    src: alfresco-content-monitored-startup.sh.j2
+    dest: "{{ binaries_folder }}/alfresco-content-monitored-startup.sh"
+    mode: 0750
+  notify:
+    - systemd-reload
+
+- name: Create monitoring start up service
+  become: true
+  template:
+    src: alfresco-content-monitored-startup.service.j2
+    dest: /etc/systemd/system/alfresco-content-monitored-startup.service
+    mode: 0644
+    owner: root
+    group: root
+  notify:
+    - systemd-reload
+
+- name: Amend alfresco-content unit
+  become: true
+  lineinfile:
```

Comment:
looks like a fast hack, can't we managed this in the alfresco-content.service template?

Reply:
I didn't do it like that because the main template is part of the tomcat role. Despite its names that unit file is generic to tomcat and would work for any webapp.
This other unit on the other hand is specific for acs and as such should not be placed in the tomcat role

Reply:
now I am wondering why a tomcat role setup a service called alfresco-content but that definitively not a problem of this PR